autonomous/rotate.py

```diff
@@ -9,5 +9,7 @@ class Rotate(AutonomousStateMachine):
 
     @state(first=True)
     def rotate(self):
-        if self.drivetrain.rotate(degrees=-90):
+        if self.drivetrain.rotate(degrees=-90):     
+            '''This tells us that if the robot rotates 90 degrees, the state is done.
```

Comment:
It's Python convention since [PEP 257](https://www.python.org/dev/peps/pep-0257/) to use double quotes rather than single quotes for docstrings. We haven't been very consistent about this, and I'm pretty sure a good amount of code I wrote uses single quotes, but it would awesome if you could change this to double quotes.